justfile

```diff
@@ -79,8 +79,12 @@ docker-build-devnet:
 
 # Start a devnet from genesis
 start-devnet:
-  docker run -it -p 26657:26657 -p 26656:26656 {{DEVNET_NAME}}:{{DEVNET_VERSION}}
+  docker run --name devnet -it -p 26657:26657 -p 26656:26656 {{DEVNET_NAME}}:{{DEVNET_VERSION}}
```

Comment:
Maybe use the chain ID as container name?

Reply:
Sure. will fix